bakerydemo/templates/base/home_page.html

```diff
@@ -26,38 +26,29 @@ <h1>{{ page.title }}</h1>
 
 <div class="container">
     <div class="row promo-row">
-        <div class="col-sm-5 promo">
-            {% if page.promo_image or page.promo_title or page.promo_text %}
-                <figure>{% image page.promo_image fill-200x200-c100 %}</figure>
-                <div class="promo-text">
-                    <h3>{{ page.promo_title }}</h3>
-                    {{ page.promo_text|safe }}
-                </div>
+        <div class="featured-cards col-sm-5 col-sm-offset-1 ">
+            {% if page.featured_section_1 %}
+                <h2 class="featured-cards__title">{{ page.featured_section_1_title }}</h2>
+                <ul class="featured-cards__list">
+                    {% for childpage in page.featured_section_1.specific.children|slice:"3" %}
+                        {% include "includes/card/listing-card.html" with title=childpage.title card_image=childpage.image %}
+                    {% endfor %}
+                </ul>
+                <a class="featured-cards__link" href="/breads">
+                    <span>View more of our breads</span>
+                    {% include "includes/chevron-icon.html" with class="featured-cards__chevron-icon" %}
```

Comment:
suggestion: aria-hide this if not already

Reply:
I've added aria-hidden="true" to the SVG component that this is including.